gradle/project.gradle.kts

```diff
@@ -211,48 +207,51 @@ tasks {
    }
 
    val kotlinc by creating(Download::class) {
+      group = "build setup"
+      val kotlincDir = dirApp/"kotlinc"
+      description = "Downloads the kotlin compiler into $kotlincDir"
+
       val os = org.gradle.internal.os.OperatingSystem.current()
       val useExperimentalKotlinc = "player.kotlinc.experimental".prjProp?.toBoolean() ?: true
-      val dirKotlinc = dirApp/"kotlinc"
-      val fileKotlinVersion = dirKotlinc/"build.txt"
-      val nameKotlinc = when {
+      val kotlinVersionFile = kotlincDir/"version"
+      val kotlinExperimentalVersion = kotlinVersion.substringBeforeLast('.')
+      val kotlincZipName = when {
          !useExperimentalKotlinc -> "kotlin-compiler-$kotlinVersion.zip"
-         os.isLinux -> "kotlin-native-linux-1.3.tar.gz\n"
```

Comment:
I hardcoded the version because we used kotlin version which did not have compiler release, looks like I forgot to change it back.

ALso, experimental compiler stopped compiling the widgets successfully for me at around 1.3.30, I will try if it works now and if not further investigate

Reply:
The fact that the experimental compiler didn't work doesn't have to do with this PR though, right?

Reply:
Correct.